Makefile

```diff
@@ -59,7 +59,7 @@ endif
 
 export GOOS ?= $(GOOS_LOCAL)
 
-RUN = docker run -t --sig-proxy=true -u $(UID) --rm \
+RUN = docker run -t -i --sig-proxy=true -u $(UID) --rm \
```

Comment:
why `-i`?

Reply:
This makes ctrl-c work correctly. Otherwise, you hit ctrl-c during docker run, and docker returns to the shell, but the continer keeps running in the background.